virtio-devices/Cargo.toml

```diff
@@ -26,9 +26,10 @@ seccompiler = "0.2.0"
 serde = "1.0.136"
```

Comment:
Maybe expand on the commit message a tad? Just explain what a VDPA device is at a high level.